src/Mvc/test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/ComplexTypeModelBinderTest.cs

```diff
@@ -602,8 +599,11 @@ public async Task BindModelAsync_ModelIsNotNull_DoesNotCallCreateModel()
             // Arrange
             var bindingContext = CreateContext(GetMetadataForType(typeof(Person)), new Person());
             var originalModel = bindingContext.Model;
+            var binders = bindingContext.ModelMetadata.Properties.ToDictionary(
+                keySelector: item => item,
+                elementSelector: item => (IModelBinder)null);
 
-            var binder = new Mock<TestableComplexTypeModelBinder>() { CallBase = true };
+            var binder = new Mock<TestableComplexTypeModelBinder>(binders) { CallBase = true };
```

Comment:
Required now because of the check at https://github.com/aspnet/AspNetCore/pull/6793/files#diff-44e4a40c4a5c7c29dd5892f664b5b362R142 (Didn't like the idea of adding another `protected virtual` method for two tests.)